typing/printpat.ml

```diff
@@ -56,17 +56,21 @@ let rec pretty_val : type k . _ -> k general_pattern -> _ = fun ppf v ->
   | Tpat_constant c -> fprintf ppf "%s" (pretty_const c)
   | Tpat_tuple vs ->
       fprintf ppf "@[(%a)@]" (pretty_vals ",") vs
-  | Tpat_construct (_, cstr, []) ->
+  | Tpat_construct (_, cstr, [], _) ->
       fprintf ppf "%s" cstr.cstr_name
-  | Tpat_construct (_, cstr, [w]) ->
+  | Tpat_construct (_, cstr, [w], None) ->
       fprintf ppf "@[<2>%s@ %a@]" cstr.cstr_name pretty_arg w
-  | Tpat_construct (_, cstr, vs) ->
+  | Tpat_construct (_, cstr, vs, vto) ->
       let name = cstr.cstr_name in
-      begin match (name, vs) with
-        ("::", [v1;v2]) ->
+      begin match (name, vs, vto) with
+        ("::", [v1;v2], _) ->
```

Comment:
Should be `None` in the new argument, this one can actually happen.

Reply:
Ok